modal/volume.py

```diff
@@ -0,0 +1,106 @@
+# Copyright Modal Labs 2023
+from typing import Optional
+
+from modal_proto import api_pb2
+from modal_utils.async_utils import synchronize_api
+from modal_utils.grpc_utils import retry_transient_errors
+
+from ._resolver import Resolver
+from .object import _Handle, _Provider
+
+
+class _VolumeHandle(_Handle, type_prefix="vo"):
+    """mdmd:hidden Handle to a `Volume` object."""
```

Comment:
Didn't spend a huge amount of effort on documentation - figure we'll revisit that once this is no longer experimental and `mdmd:hidden` is removed.